### PR TITLE
proovframe map init push

### PR DESCRIPTION
**Pull request by vagkaratzas**
## PR checklist

Closes #XXX 

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - For modules:
    - [ ] `nf-core modules test <MODULE> --profile docker`
    - [ ] `nf-core modules test <MODULE> --profile singularity`
    - [ ] `nf-core modules test <MODULE> --profile conda`
  - For subworkflows:
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile docker`
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile singularity`
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile conda`

### CODE REVIEW

File: modules/nf-core/proovframe/map/meta.yml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+      description: "frame-shift correction for long read (meta)genomics"
+      homepage: "https://github.com/thackl/proovframe"
+      documentation: "https://github.com/thackl/proovframe"
+      tool_dev_url: "https://github.com/thackl/proovframe"
+      doi: "10.1101/2021.08.23.457338"
+      licence: ["MIT"]
+      identifier: biotools:proovframe
+
+input:
+  # Only when we have meta
```
Comment (sateeshperi) [inline modules/nf-core/proovframe/map/meta.yml:20]:
```suggestion
```

File: modules/nf-core/proovframe/map/meta.yml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  - - meta:
+        type: map
+        description: |
+          Groovy Map containing sample information
+          e.g. `[ id:'sample1', single_end:false ]`
+
```
Comment (sateeshperi) [inline modules/nf-core/proovframe/map/meta.yml:26]:
```suggestion
```

File: modules/nf-core/proovframe/map/meta.yml [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+    - faa:
+        type: file
+        description: A FASTA file containing a database of guide protein sequences
+        pattern: "*.{faa,fasta,fa}"
+
```
Comment (sateeshperi) [inline modules/nf-core/proovframe/map/meta.yml:31]:
```suggestion
```

File: modules/nf-core/proovframe/map/meta.yml (hunk 1 above)
Comment (sateeshperi) [reply modules/nf-core/proovframe/map/meta.yml:20]: is there a case where there will be no meta ?

File: modules/nf-core/proovframe/map/meta.yml [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+      - meta:
+          type: map
+          description: |
+            Groovy Map containing sample information
+            e.g. `[ id:'sample1', single_end:false ]`
+      - "*.tsv":
+          type: file
+          description: Output TSV file with the frameshift-aware protein to read alignments
+          pattern: "*.tsv"
+
```
Comment (sateeshperi) [inline modules/nf-core/proovframe/map/meta.yml:49]:
```suggestion
```

**Review by sateeshperi — APPROVED** [review]
minor comments

File: modules/nf-core/proovframe/map/meta.yml (hunk 1 above)
Comment (vagkaratzas) [reply modules/nf-core/proovframe/map/meta.yml:20]: Naah, it was just there from the tempalte. Removed!

**Pull request merged**